### ISSUE TITLE
Text goes out of border

### ISSUE THREAD

**Issue by rEnr3n** [OllieTerrance/Homely#7]
I've set a long text for a link and the text goes out of border. Please make the buttons/borders self-adjust.

**Comment by OllieTerrance** [comment]
Does adding the following to your custom CSS fix it?

```css
#links .btn {
  white-space: normal;
}
```

**Comment by rEnr3n** [comment]
Yes. Thank you.

**Comment by Droyk** [comment]
@Terrance Instead of this CSS workaround. can't you just add this functionality in the extension itself? as I mentioned this on my thread #11 I work with too many new computers adding this CSS snippet workaround every time is a bit of a hassle. so just for the sake of convenience & also for the sake of new users who don't know about this CSS workaround can you **Please update this extension with this CSS workaround!!**

**Comment by Terrance** [comment]
> can't you just add this functionality in the extension itself?

It was, in bbf130b, but apparently that never made it into a release.  😐  Been a while since I've packaged a release, I'll see if I can get an update out.

**Comment by Droyk** [comment]
[Please do it.](https://www.youtube.com/watch?v=ZXsQAXx_ao0)

**Comment by Terrance** [comment]
Released 1.7.2, hopefully that'll become available on the Chrome Web Store soon.

**Comment by Droyk** [comment]
Got it. Thank you, by the way, why did you stop working on it? your last update was in 2016 never saw a good update after that. I mean no new features, no bug fixes, no code enhancements, nothing. why? this is the best new tab I have used to date in the terms of sites management, simplicity & a feature-packed new tab experience no extension I have used in past comes near it. 

**I would love to see more from you.**
